documentation/jetty-documentation/src/main/asciidoc/operations-guide/server/server-logging-server.adoc

```diff
@@ -173,28 +165,34 @@ $JETTY_BASE
+Jetty downloaded the required Log4j2 `+*.jar+` files, and created a `$JETTY_BASE/resources/log4j2.xml` file that you can configure to customize your Log4j2 logging.
+
+Please refer to the link:https://logging.apache.org/log4j/2.x/manual/configuration.html[Log4j2 configuration manual] for more information about how to configure Log4j2.
 
 [[og-server-logging-server-bridges]]
 ====== Bridging Logging to SLF4J
 
 When you use libraries that provide the features you need (for example, JDBC drivers), it may be possible that those libraries use a different logging framework than SLF4J.
 
 SLF4J provides link:http://www.slf4j.org/legacy.html[bridges for legacy logging APIs] that allows you to bridge logging from one of these legacy logging frameworks to SLF4J.
-Once the logging is bridged to SLF4J, you can use the xref:og-server-logging-server-default[default configuration] or the xref:og-server-logging-server-custom[custom configuration] so that your logging is centralized in one place only.
+Once the logging is bridged to SLF4J, you can use Jetty's xref:og-server-logging-server-default[default configuration] or a xref:og-server-logging-server-custom[custom configuration] so that your logging is centralized in one place.
+
+Jetty provides several modules out of the box for bridging older logging frameworks to SLF4J:
 
-Jetty provides out-of-the-box modules that you can enable to bridge logging from other logging frameworks to SLF4J.
+* `logging-jul-capture` for bridging from `java.util.logging`
+* `logging-jcl-capture` for bridging from Jakarta Commons Logging (JCL)
+* `logging-log4j1-capture` for bridging from Apache Log4j
 
 [[og-server-logging-server-bridge-jul]]
-====== Bridging `java.util.logging`
+====== Bridging from `java.util.logging`
 
-For libraries that use `java.util.logging` as their logging framework you can enable the `logging-jul-capture` Jetty module.
+For libraries that use `java.util.logging` as their logging framework, you can enable Jetty's `logging-jul-capture` module:
 
 ----
 $ java -jar $JETTY_HOME/start.jar --add-modules=logging-jul-capture
 ----
 
-The `logging-jul-capture` Jetty module implies `--exec` and therefore spawns a second JVM (see xref:og-start-start[this section]) because it needs to provide the system property `java.util.logging.config.file` (so that `java.util.logging` can read the configuration from the specified file), and because it needs to make available on the System ClassLoader the class `org.slf4j.bridge.SLF4JBridgeHandler`.
+The `logging-jul-capture` module implies `--exec` and therefore xref:og-start-start[spawns a second JVM] because it needs to provide the system property `java.util.logging.config.file` (so that `java.util.logging` can read the configuration from the specified file), and because it needs to make available on the System ClassLoader the class `org.slf4j.bridge.SLF4JBridgeHandler`.
```

Comment:
This section gets into a level of detail that, while useful and interesting, may be a little esoteric for the average user interested in figuring out logging in Jetty. Perhaps we can move this content somewhere else?

Here's one thought: This section on logging is effectively documenting several different modules, so perhaps we can move the more detailed content into the Jetty Modules section of the documentation by creating new sub-sections for each of the individual logging modules.

Reply:
We should have a section somewhere in the documentation that already describes the `--exec` option.  So in logging, all we need do is have a note that says something like: Since JUL logging needs to be configured on the JVM command line, this module implies the --exec option (with link to other documentation)

Reply:
@gregpoulos my take is that these sections need to exist, be very short, and point to the module documentation, but they do need to exist here because they explain how to configure the server logging.

Like you suggest I would move this paragraph to a more detailed documentation of the `logging-jul-capture` module, but keep here the section about bridging to `jul` (like we have the one for Log4j2).

@gregw we do have a section about `--exec` already.

Reply:
I think I can get rid of this `--exec` (actually `[exec]`) configuration in `logging-jul.mod` and `logging-jul-capture.mod` entirely. (thanks to some niceties that Java 17 gives us)
That would make no logger configuration we ship have an exec requirement.
I'll give it a whirl when I get back (at the end of the month).

Reply:
@sbordet, I'll hold off on making changes here in light of @joakime's comment.

Reply:
I would say keep the section for now and let's move on with this PR.
When Joakim reviews the need for --exec, we can update this section.